requirements.txt

```diff
@@ -1,8 +1,9 @@
-abjad
-numpy
-tqdm
-nltk
-pillow
-pytorch-lightning
-torch
-transformers
\ No newline at end of file
+abjad==3.4
+nltk==3.7
+numpy==1.23.3
+Pillow==9.4.0
+pytorch_lightning==1.8.1
+torch==1.13.1
+tqdm==4.64.0
+transformers==4.25.1
+sentencepiece
```

Comment:
Why is there no version for sentencepiece?

Reply:
added it to the transformers line now